docs/intro/contributing.txt

```diff
@@ -282,7 +282,7 @@ We'll now implement this feature and associated tests.
 Creating a branch
 =================
 
-Before making any changes, create a new branch for the ticket:
```

Comment:
Thanks for this patch, however "must" is unnecessary. We try to avoid unnecessary changes as they create extra work for translators.